build/universal/trending/english.yaml

```
@@ -1,60 +1,60 @@
+article9title: "Nested Lists in Python"
+article9link: "https://www.freecodecamp.org/news/list-within-a-list-in-python-initialize-a-nested-list/"
+article10title: "Coalesce SQL"
+article10link: "https://www.freecodecamp.org/news/coalesce-sql-example-postgresql-and-sql-server-functions/"
+article11title: "Python join()"
+article11link: "https://www.freecodecamp.org/news/python-join-how-to-combine-a-list-into-a-string-in-python/"
+article12title: "JS Post Request"
+article12link: "https://www.freecodecamp.org/news/javascript-post-request-how-to-send-an-http-post-request-in-js/"
+article13title: "JS Type Checking"
+article13link: "https://www.freecodecamp.org/news/javascript-type-checking-how-to-check-type-in-js-with-typeof/"
+article14title: "Read Python File"
+article14link: "https://www.freecodecamp.org/news/how-to-read-a-file-line-by-line-in-python/"
+article15title: "SOLID Principles"
+article15link: "https://www.freecodecamp.org/news/solid-design-principles-in-software-development/"
+article16title: "Sort a List in Java"
+article16link: "https://www.freecodecamp.org/news/how-to-sort-a-list-in-java/"
+article17title: "For Loops in Python"
+article17link: "https://www.freecodecamp.org/news/for-loops-in-python-with-example-code/"
+article18title: "JavaScript 2D Array"
+article18link: "https://www.freecodecamp.org/news/javascript-2d-arrays/"
+article19title: "SQL CONVERT Function"
+article19link: "https://www.freecodecamp.org/news/sql-convert-the-date-to-string-or-datetime-function-2/"
+article20title: "Rename Column in Pandas"
+article20link: "https://www.freecodecamp.org/news/how-to-rename-a-column-in-pandas/"
+article21title: "Delete a File in Python"
+article21link: "https://www.freecodecamp.org/news/how-to-delete-a-file-in-python-and-remove-a-directory-too/"
+article22title: "K-Nearest Neighbors Algo"
+article22link: "https://www.freecodecamp.org/news/k-nearest-neighbors-algorithm-classifiers-and-model-example/"
+article23title: "iferror Function in Excel"
+article23link: "https://www.freecodecamp.org/news/iferror-function-in-excel-example/"
+article24title: "Remove from String Python"
+article24link: "https://www.freecodecamp.org/news/remove-from-string-in-python-how-to-remove-characters-from-a-string/"
+article25title: "Create a File in Terminal"
+article25link: "https://www.freecodecamp.org/news/how-to-make-a-file-in-linux-from-the-command-line-create-a-file-in-terminal/"
+article26title: "Clear Formatting in Excel"
+article26link: "https://www.freecodecamp.org/news/how-to-clear-formatting-in-excel/"
+article27title: "Accounting Num Format Excel"
+article27link: "https://www.freecodecamp.org/news/accounting-number-format-in-excel-how-to-apply-it-to-selected-cells/"
+article28title: "Check if File Exists Python"
+article28link: "https://www.freecodecamp.org/news/how-to-check-if-a-file-exists-in-python/"
+article29title: "Iterate Over Dict in Python"
+article29link: "https://www.freecodecamp.org/news/dictionary-iteration-in-python/"
```

This should add the newline back to the end of the file and fix the lint test:

```suggestion
article29link: "https://www.freecodecamp.org/news/dictionary-iteration-in-python/"

```